comment.py

```diff
@@ -14,7 +14,7 @@
     agent = CommentAgent(
         repo_name=REPO_NAME,
         branch_or_prnum=BRANCH_OR_PR_NUMBER,
-        branch=MAIN_BRANCH,
+        main_branch=MAIN_BRANCH,
```

Comment:
Whoa there, Nelly! The parameter switch from `branch` to `main_branch` is as unnecessary as a screen door on a submarine. The larger the footprint of a change, the more like quicksand it gets. Keep it small, keep it safe. No need for the switcheroo if it ain't broken!